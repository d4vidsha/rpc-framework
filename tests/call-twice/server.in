init 6001
register add2 add2
serve
