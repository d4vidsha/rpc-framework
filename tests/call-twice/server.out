rpc_init_server: instance 0, port 6001
rpc_register: instance 0, add2 (handler) as add2
rpc_serve_all: instance 0
handler add2_i8: arguments 127 and 127
handler add2_i8: arguments -127 and -127
